ValetWatchTest Extension/InterfaceController.m

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
Ok , the problem is not have unit testing framework for Apple Watch. 

I'm studying the possibilities, but really complicated to do all the tests in hand. More 1 or 2 days.